### PR TITLE
MNT-22155 : [Security] Share XSS via unauthenticated Surf API

### PR DESCRIPTION
**Pull request by alexbalmus**
- Attempt fixing the XSS issue by setting <authentication>admin</authentication> in several web-scripts descriptors

### CODE REVIEW

**Review by flv22 — APPROVED** [review]

**Review by araschitor — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
       script: travis_retry mvn test
     - stage: release
       name: "Push to Nexus"
-      if: fork = false AND (branch =~ /support\/.*/) AND type != pull_request AND commit_message !~ /\[no-release\]/
+      if: fork = false AND (branch =~ /support\/.*/ AND type != pull_request AND commit_message !~ /\[no-release\]/
```
Comment (pieCit87) [inline .travis.yml]: error. add the parentesis

**Review by pieCit87 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (flv22) [inline .travis.yml]: I think there is a missing closing parenthesis after branch =~ /support\/.*/

Comment (araschitor) [reply .travis.yml]: Done.
Thanks :)

Comment (araschitor) [reply .travis.yml]: Fixed :)
Thanks.

Comment (pieCit87) [reply .travis.yml]: good

**Review by pieCit87 — APPROVED** [review]
LGTM

**Review by pieCit87 — CHANGES_REQUESTED** [review]
not okay

File: .travis.yml [hunk 2]
```diff
@@ -27,19 +27,19 @@ stages:
...
+#      script:
+#        # Download the latest version of WhiteSource Unified Agent
+#        - curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+#        # Run WhiteSource Unified Agent
+#        - java -jar wss-unified-agent.jar -apiKey ${WHITESOURCE_API_KEY} -c _ci/.wss-unified-agent.config
     - name: "Source Clear Scan (SCA)"
       stage: build
       # only on SP branches or master and if it is not a PR
-      if: fork = false AND (branch = develop OR branch =~ /support\/SP\/.*/ OR branch =~ /^APPS-.*$/) AND type != pull_request
+      if: fork = false AND (branch =~ /support\/.*/) AND type != pull_request AND commit_message !~ /\[no-release\]/
```
Comment (pieCit87) [inline .travis.yml:42]: so, we execute sourceclear agent just:
1-when is not a fork
2-when branch is support/something
3-when type is not pull_request
4-when commit message doesn't contain no-release
On point 4 i think we dont need it. We should execute source clear always, you can remove the last condition

**Review by pieCit87 — CHANGES_REQUESTED** [review]

**Pull request merged**